.ls-lint.yml

```diff
@@ -0,0 +1,25 @@
+ls:
+  .dir: regex:([.a-z0-9]*)([-.][a-z0-9]+)*
```

Comment:
now you can avoid it 

https://github.com/what1s1ove/whatislove.dev/blob/main/.ls-lint.yml

Reply:
This regexp matches kebab-case but also allows dots. I can change it for the .dir, but then we will need to override this for some folders like .github, and for files we will need to leave it because of .helper etc. extentions

Reply:
updated config